pkg/workspace/state/service.go

```
@@ -34,14 +34,15 @@ func (s *Service) InitializeDaemon() (err error) {
 		return err
 	}
 
-	manifold.Walk(s.Root, func(n manifold.Object) {
+	manifold.Walk(s.Root, func(n manifold.Object) error {
 		for _, com := range n.Components() {
 			if initializer, ok := com.Pointer().(initializer); ok {
 				if err := initializer.Initialize(); err != nil {
 					log.Print(err)
 				}
 			}
 		}
+		return nil
```

Another spot to potentially return an error instead of just logging them.